src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -311,7 +322,7 @@ export const ContactPolicy = InjectAppServices(
           </div>
         </section>
         <Modal isOpen={modalIsOpen} handleClose={() => toggleModal(false)}>
-          <SubscriberListSelector />
+          <SubscriberListSelector preSelected={selectedLists} />
```

Comment:
What do you think about `selectedLists` prop instead `preSelected` prop?

Reply:
or just `preselected` ...

Reply:
`preselected` sounds good!